patches/components-bookmarks-browser-bookmark_model.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
+index ae1ef2997adabedb362c525b58c724a9114899b9..316d4db082bb65b58fec60d1fc288f340d3062d5 100644
+--- a/components/bookmarks/browser/bookmark_model.h
++++ b/components/bookmarks/browser/bookmark_model.h
+@@ -330,6 +330,7 @@ class BookmarkModel : public BookmarkUndoProvider,
+   friend class BookmarkStorage;
+   friend class ScopedGroupBookmarkActions;
+   friend class TestBookmarkClient;
++  BRAVE_FRIEND_CLASS;
```

Comment:
The macro should be `BRAVE_BOOKMARK_MODEL_H` and insert before `private:`
see `BRAVE_BROWSER_H`

Reply:
thanks, fixed